tests/e2e/process_cl61d/tests.py

```diff
@@ -0,0 +1,62 @@
+import netCDF4
+from os import path
+import pytest
+from test_utils.utils import parse_args, count_strings
+
+SCRIPT_PATH = path.dirname(path.realpath(__file__))
+
+
+class Test:
+
+    instrument = 'cl61d'
+
+    @pytest.fixture(autouse=True)
+    def _fetch_params(self, params):
+        self.full_path = params['full_path']
+        self.site, self.date, self.product = parse_args(params['args'])
+        self.date_short = self.date.replace('-', '')
+
+    def test_product_file(self):
```

Comment:
Tuote on vissiin testattu tarkemmin cloudnetpyn puolella?

Reply:
Joo, siellä on paremmat testit noille luoduille tiedostoille. Näissä e2e testeissä voi tietty testata samoja asioita, mutta tärkeintä on nähdä että fileet tulevat niin kuin pitää ja myös että `data-processing`in asettama metadata tulee tiedostoihin oikein.